crossplane.yaml

```diff
@@ -39,4 +39,4 @@ spec:
     - provider: xpkg.upbound.io/upbound/provider-azure
       version: ">=v0.13.0-0"
     - provider: xpkg.upbound.io/crossplane/provider-helm
-      version: ">=v0.10.0-0"
+      version: ">=v0.11.0-0"
```

Comment:
seems like a good idea to me - is provider-helm going to be published to marketplace soon? i only see v0.10.0 there: https://marketplace.upbound.io/providers/crossplane/provider-helm/

Reply:
It doesn't seem like publishing to marketplace is integrated yet, [this is the CI run](https://github.com/crossplane-contrib/provider-helm/actions/runs/3113607912/jobs/5048454425). @hasheddan How was the v0.10.0 published? Should I build locally and push? And can we add Upbound tokens to this repo, I don't seem to have permissions.

Reply:
And thanks for catching @jbw976 !

Reply:
I opened https://github.com/crossplane-contrib/provider-helm/pull/142 to add xpkg publishing to provider-helm but it needs credentials and I have neither Github repo permissions to add them nor Upbound permissions on `xpkg.upbound.io/crossplane` to get them.

Reply:
provider-helm is [published](https://marketplace.upbound.io/providers/crossplane/provider-helm/v0.11.1) and I updated both versions!